server/asset/btc/btc.go

```diff
@@ -219,12 +197,14 @@ func (btc *Backend) Connect(ctx context.Context) (*sync.WaitGroup, error) {
 		return nil, fmt.Errorf("error creating %q RPC client: %w", btc.name, err)
 	}
 
-	// Setting the client field will enable shutdown
-	btc.client = client
-	btc.node = client
+	// Requester must be set otherwise there can be a panic on shutdown.
```

Comment:
> there can be a panic on shutdown

If you're talking about `(*Backend).shutdown`, then I don't think that's true, since you check `btc.node != nil`